app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
```

Comment:
yeah. I guess classes are better. Lets use a class, and maybe we can re-work the other notifier to change for a class too in the future